tap_chargebee/schemas/gifts.json

```diff
@@ -0,0 +1,79 @@
+{
+	"type": ["null", "object"],
+	"properties": {
+		"id": {
+			"type": ["null", "string"]
+		},
+		"status": {
+			"type": ["null", "string"]
+		},
+		"scheduled_at": {
+
+			"type": ["null", "string"],
+			"format": "date-time"
+		},
+		"auto_claim": {
+			"type": ["null", "boolean"]
+		},
+		"claim_expiry_date": {
+			"type": ["null", "string"],
+			"format": "date-time"
+		},
+		"resource_version": {
+			"type": ["null", "integer"]
+		},
+		"updated_at": {
+			"type": ["null", "string"],
+			"format": "date-time"
+		},
+		"gifter": {
+			"type": ["null", "object"],
+			"properties": {
+				"customer_id": {
+					"type": ["null", "string"]
+				},
+				"invoice_id": {
+					"type": ["null", "string"]
+				},
+				"signature": {
+					"type": ["null", "string"]
+				},
+				"note": {
+					"type": ["null", "string"]
+				}
+			}
+		},
+		"gift_receiver": {
+			"type": ["null", "object"],
+			"properties": {
+				"customer_id": {
+					"type": ["null", "string"]
+				},
+				"subscription_id": {
+					"type": ["null", "string"]
+				},
+				"first_name": {
+					"type": ["null", "string"]
+				},
+				"last_name": {
+					"type": ["null", "string"]
+				},
+				"email": {
+					"type": ["null", "string"]
+				}
+			}
+		},
+		"gift_timelines": {
+			"type": ["null", "array"],
+			"properties": {
```

Comment:
I get an error here as I believe that this `"properties"` key wants to be nested in an `"items"` block.